cloudbuild.yaml

```diff
@@ -293,7 +292,7 @@ steps:
     path: /go
   id: 'compat-report'
 
-timeout: 600s
+timeout: 900s
```

Comment:
wow, cloudbuild is taking more than 10 minutes?  That is too slow for a CI check.  @bstoll is this latency  expected?

Reply:
The latency is from pyangbind's code generator trying to generate all the code in OC. Given it's python I think it makes sense.

The generation is already being done in parallel:
https://github.com/openconfig/models-ci/blob/3f6a71eba144cc3aa4b72ab5110e5064644d8b9d/cmd_gen/main.go#L301C14-L301C14

Reply:
Alright, for some reason if I use a local version of pyangbind instead of the one downloaded into the Python virtual env cache for pyang's `--plugindir` it halves the time.

Total build time is now 7min

Reply:
I will now keep pyangbind running against its main branch instead of what's published on pip